src/libstd/lib.rs

```diff
@@ -12,33 +12,35 @@
 //!
 //! The Rust Standard Library provides the essential runtime
 //! functionality for building portable Rust software.
-//! It is linked to all Rust crates by default.
 //!
-//! ## Intrinsic types and operations
+//! It is linked to all Rust crates by default as though they
```

Comment:
> The rust standard library is available to all rust crates by default, just as if... 

Small wording suggestion that I think makes this more newbie friendly :)

Also, as a newbie, I don't quite understand what "crate-level" means in this context? As far as my understanding goes, there's no way to automatically have `extern crate some_crate` for all modules in a crate (besides the std library), but its very possible I am mistaken.